src/server/scripts/Spells/spell_item.cpp

```diff
@@ -3729,21 +3729,22 @@ class spell_item_mirrens_drinking_hat : public SpellScript
 
     void HandleScriptEffect(SpellEffIndex effIndex)
```

Comment:
effIndex is unused, please comment out the parameter (Travis and Circle CI report build warnings)